test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -68,6 +71,22 @@ class RedirectBindingVerifier(private val response: IdpRedirectResponseDecorator
         }
     }
 
+    /**
+     * Verifies the http status code of the response according to the redirect binding rules in the binding spec
+     * 3.4.6 Error Reporting
+     */
+    private fun verifyHttpStatusCode() {
+        // TODO remove the 200 check when "Manually change DDF IdP to respond with 302/303 status code for Redirect" is completed
+        if (response.httpStatusCode != 200 && response.httpStatusCode != 302 && response.httpStatusCode != 303) {
+            SAMLComplianceException.createWithPropertyNotEqualMessage(
+                    SAMLSpecRefMessage.SAMLBindings_3_4_6_a,
+                    property = "HTTP Status Code",
+                    propertyValue = response.httpStatusCode.toString(),
+                    otherValue = HttpStatusCodes.STATUS_CODE_OK.toString()
```

Comment:
✏️ This `otherValue` doesn't seem correct when we're expecting redirect codes.

Reply:
I'll format it into an intuitive string (using the proper `HTTPStatusCodes` object):
`otherValue = "302 or 303"`